toonz/sources/toonzfarm/tfarmcontroller/tfarmcontroller.cpp

```diff
@@ -90,14 +97,16 @@ TFilePath getLocalRoot() {
   TFilePath lroot;
 
 #ifdef _WIN32
-  TFilePath name(L"SOFTWARE\\OpenToonz\\OpenToonz\\1.0\\TOONZROOT");
+  QString regpath = QString::fromStdString("SOFTWARE\\OpenToonz\\OpenToonz\\" + std::string(applicationVersion) + "\\FARMROOT");
```

Comment:
@yamisama (Peter?) just found your response,

Thanks for this I actually learned a lot from it, so it was quite helpful. I am new to c++ so still trying to figure things out, as I mostly code in ancient (ansii 99) C or python, though I have worked in c# and a bunch of other languages as well in the past, so mostly it's the C++ specific bits i need to get familiar with.

The build system sorted issue seems to be sorted and the version control stuff i'll figure out eventually even if just through trial and error.

Thanks for your help.

Dave.